docs/superpowers/plans/2026-03-23-test-coverage-plan.md

```
@@ -0,0 +1,1611 @@
+  return openMemoryDatabase();
+}
+```
+
+- [ ] **Step 5: Verify infrastructure works**
+
+Run: `cd apps/desktop && npx vitest run --passWithNoTests`
+Expected: Vitest runs successfully with 0 tests.
+
+- [ ] **Step 6: Commit**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)